vendor/woocommerce/action-scheduler/classes/WP_CLI/Action/Cancel_Command.php

```
@@ -0,0 +1,120 @@
+	 * @param string $group The group the job is assigned to.
+	 * @return void
+	 */
+	protected function cancel_all( $hook, $callback_args, $group ) {
+		if ( empty( $hook ) && empty( $group ) ) {
+			\WP_CLI::error( __( 'Please specify hook and/or group of actions to cancel.', 'action-scheduler' ) );
+		}
+
+		try {
+			$result = as_unschedule_all_actions( $hook, $callback_args, $group );
+		} catch ( \Exception $e ) {
+			$this->print_error( $e, $multiple );
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$multiple'. (undefined)

(UnusedLocalVariable)